src/components/Post.js

```diff
@@ -10,189 +11,118 @@ import FormControlLabel from '@mui/material/FormControlLabel';
-                        //borderBottom: 1 
-                     }}
+    return (
+        <div>
+            <Grid
+                container
+                direction="column"
+                justifyContent="center"
                     >
-                        <div >
+                <Grid item
+                    bgcolor =   '#e0e0e0'
+                    alignItems ='center s'>   
+                    <div style = {heading}>Create A New Post</div>
+                </Grid>
 
-                        <TextField
-                            style={{ marginTop: 5 ,
-                            marginBottom: 5,
-                            width: '50%',
-                            //height: '90%',
-                        
-                            }}
-                            id="addTitle"
-                            label="title"
-                            variant="filled"
-                            value = {this.state.title} 
-                            onChange = {this.handleTiltle}
-                            />
-                        
-                        </div>
-                     </Box>
+                <Grid item  
+                    bgcolor = '#eeeeee'
+                    alignItems = 'flex-start'>
+                    <TextField
+                        style={{ marginTop: 5 ,
+                        marginBottom: 5,
+                        width: '50%',
+                        marginLeft: 5 }}
+                        id="addTitle"
+                        label="title"
+                        variant="filled"/>
+                </Grid>
 
-                     <Box
-                        //borderRadius={3}
-                        sx={{
-                        p: 5,    
-                        
-                        bgcolor:  '#e0e0e0',
-                        alignItems:'flex-start',
-                        //borderBottom: 1 
-                     }}>
-                        <div >
+                <Grid item
+                    bgcolor =  '#e0e0e0' 
+                    alignItems = 'flex-start'>
+
+                    <TextField
+                        style={{ marginTop: 5 ,
+                        marginBottom: 5,
+                        width: '90%',
+                        height: '90%',
+                        marginLeft: 5 }}
+                        id="addDescription"
+                        label="add description"
+                        variant="filled"
+                        multiline
+                        rows={10}
+                        //onChange = {this.handleCommon} 
+                        />         
+                </Grid>
 
-                        <TextField
-                            style={{ marginTop: 5 ,
-                            marginBottom: 5,
-                            width: '90%',
-                            height: '90%',
+                <Grid  item
+                    bgcolor = '#eeeeee'
```

Comment:
I don't think bgcolor is necessary

Reply:
ok

Reply:
please remove that as well

Reply:
please commit. and push your code after you made change so that I can see what u change

Reply:
And just a little tip: using the makeStyles, instead of hardcoding the CSS to sx fields of every HTML label, might make your life easier. 
For more details, you can look at the styleClasses of Profile, NavMenu, and PostStream.
The other parts LGTM.